Header header
int32[] ids
ar_target/ARMarker3d[] markers
sensor_msgs/PointCloud2 points
sensor_msgs/Image left_image
sensor_msgs/Image right_image
sensor_msgs/CameraInfo left_camera_info
sensor_msgs/CameraInfo right_camera_info
geometry_msgs/Transform head_transform_base
geometry_msgs/Transform cam_transform_head
